src/todo-list-kata/day_01/__tests__/todo-list.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { shallow, configure } from 'enzyme';
+import { should } from 'chai';
+import Adapter from 'enzyme-adapter-react-16';
+
+import TodoList from '../kata';
+
+configure({ adapter: new Adapter() });
+should();
+
+describe('todo list', () => {
+  it('renders an empty todo list', () => {
+    const list = shallow(<TodoList/>);
+
+    list.find('div[id="todo-list"]').should.exist;
```

Comment:
I think, it less obvious but easier. To make it more obvious, better ID name could be chosen

```suggestion
    list.find('#todo-list').should.exist;
```